cluster-provision/k8s/1.17/provision.sh

```diff
@@ -121,9 +121,49 @@ echo "net.netfilter.nf_conntrack_max=1000000" >> /etc/sysctl.conf
 
 systemctl restart NetworkManager
 
+mkdir -p /tmp/kubeadm-patches/
+
+cat >/tmp/kubeadm-patches/kustomization.yaml <<EOF
+patchesJson6902:
+- target:
+    version: v1
+    kind: Pod
+    name: kube-apiserver
+    namespace: kube-system
+  path: add-security-context.yaml
+- target:
+    version: v1
+    kind: Pod
+    name: kube-controller-manager
+    namespace: kube-system
+  path: add-security-context.yaml
+- target:
+    version: v1
+    kind: Pod
+    name: kube-scheduler
+    namespace: kube-system
+  path: add-security-context.yaml
+- target:
+    version: v1
+    kind: Pod
+    name: etcd
+    namespace: kube-system
+  path: add-security-context.yaml
+EOF
```

Comment:
These 4 pods already seem to be running as spc_t in 1.17 without this (not true for 1.16 though).
What am I missing?

Reply:
hmm.. didn't see it there, but I'll double check.

Reply:
Re-testing on my side too, maybe I got confused with something else.

Reply:
I only see `securityContext: {}` by default.
Actually that may be the reason why `kustomize` didn't work. I should probably do `replace` instead of `add`

Reply:
My bad, the pods were running as spc_t because my image didn't include https://github.com/kubevirt/kubevirtci/pull/311

Reply:
This is still a lot of pods running unconfined. Did you make sure each one of them needs to run as spc_t? We should probably keep this list to a minimum and maybe file a ticket to try and fix the remaining ones.

Reply:
yea, the control plane pods fail without spc_t :/
I think we should start somewhere and then try to remove it one it works.

Reply:
Fr om my perspective we explicitly don't have to care about the selinuxcontext of the k8s control plane. No need to invest time into that. Great work.